test/src/tests-batchUploader.ts

```diff
@@ -671,7 +672,7 @@ describe('batch uploader', () => {
             done();
         });
 
-        it('should purge events from Session Storage upon Batch Creation', (done) => {
+         it('should purge events from Session Storage upon Batch Creation', (done) => {
```

Comment:
For some reason, there's a stray space here.

```suggestion
        it('should purge events from Session Storage upon Batch Creation', (done) => {
```